contents/handbook/strategy/overview.md

```diff
@@ -140,24 +142,37 @@ While there is value in the items on the right, we value the items on the left m
-    * **Customers**
-        * **Focus on high-growth startups.** E.g. Large initial contracts ($20k-$70k/year) and smaller deals in organizations that will eventually become large
-        * **Non Goal:** Start doing outbound sales
-    * **Product**
-        * **Goals:**
-            * **Quality:** Our core features (insights, recordings, feature-flags / experimentation) work like a Swiss watch
-            * **Self Service Subscription:** Customers of every size sign-up, start using and subscribe to PostHog without asking us for help
-        * **Non Goals:**
-            * Build lots of new low quality / partial features
-            * Build everything a single enterprise customer wants just to close a deal
-
-## Target customers for 2022
-
-We build for the highest-performing **product teams** (engineers, PMs, designers) building the **most loved products** at **high-growth startups**. We focus specifically on the  **product-engineers** (full-stack engineers skewed towards the frontend) but it should be usable by everyone in the product team.
+## Direction for 2023
+
+3-word summary: **Nail product engineering**
+
+* **Customers**
+  * **Focus on product engineers at high-growth startups.** E.g. Large initial contracts ($20k-$70k/year) and smaller startups that will eventually become large
+  * **Non Goal:** Start doing outbound sales; build products that only impact marketing/sales/CS teams
+* **Product**
+  * **Goals:**
+    * 10x the core UX for product engineers:
+      * PostHog 3000 UX = a design and UX uplift including dark mode
+        * Why? Re-classify PostHog from a PM tool to a developer tool to better resonate with our audience. Reduce the friction in using multiple tools
+      * More powerful querying
+        * Why? Enable answering the long tail of questions that engineers have about their data
+      * Make it easy to evaluate the success of features
+        * Why? Currently, it's manual and repetitive to do best practice evaluation of features. Or it's not possible e.g. we don't have session recordings on mobile
+    * Launch the CDP as a full product
+      * Why?
+        * Putting PostHog at the center of the stack gets all the data into PostHog, enabling our customers to ask the questions they need about their data
+        * Makes PostHog incredibly sticky even if they move to a data warehouse
+        * Our customers are already asking for it
+      * How?
+        * Make imports and exports bulletproof, 1st class webhook destinations, build or integrate with key sources and destinations
+  * **Non Goals:**
+    * Build lots of new low-quality / partial features
+    * Build everything a single enterprise customer wants just to close a deal
+
+## Target customers for 2023
+
+We build for the **product-engineers** (full-stack engineers skewed towards the frontend) building the **most loved products** at **high-growth startups**. But it should be usable by everyone in the product team (engineers, PMs, designers).
```

Comment:
Sorry I know this was already on the ICP page in the handbook, but I don't think the definition of 'product engineer' as a 'full stack engineer skewed towards the frontend' is the most useful one. 

I appreciate that code they write is typically frontend-focused, but what actually characterizes them to me is [much more expansive](https://posthog.com/blog/what-is-a-product-engineer) (e.g. own features, talk to users, make design decisions).

Reply:
Great point, I've linked to that article